setup.py

```diff
@@ -89,6 +89,8 @@ def _get_readme():
                         'requests>=2.4.1,<3.0',
                         'lxml>=3.6.0,<4.0',
                         'click>=6.0,<7.0'
+                        'prettytable>=0.7.2',
+                        'PTable>=0.9.2',
```

Comment:
do we still need Ptable?

Reply:
No we don't need it anymore. It was to include titles. Since we don't want them anymore, I can remove it from my code and push the commit.

Reply:
okay, I'll approve this pr once you remove it. Also, you need to update this branch (rebase, or merge master into it).